scripts/thickclient/arm_wrapper.sh

```diff
@@ -4,7 +4,7 @@ set -eo pipefail
 
 DEVNAME=$1
 PROTECTION=""
-USER="arm"
+USER="mtech"
```

Comment:
Please revert this change

Reply:
again. slipped through to the keeper
will fix